### ISSUE TITLE
NVDA does not read link text when hovering with the mouse in Chromium-based browsers v140

### ISSUE THREAD

**Issue by albzan** [nvaccess/nvda#18996]
### Brief summary

When using NVDA with a Chromium-based browser (Google Chrome or Microsoft Edge) version 140, NVDA does not announce the text when hovering the mouse pointer over a link element.

### Steps to reproduce

1. Launch Microsoft Edge v140.x or Google Chrome v140.x.
2. Start NVDA with Mouse tracking enabled.
3. Open any webpage that contains links.
4. Hover the mouse pointer over a link.

### Actual behavior

NVDA does not announce anything when hovering the mouse over a link.

### Expected behavior

NVDA should announce the link text when the mouse pointer hovers over it.

### NVDA type

portable copy

### NVDA version

2025.3

### Have you tried any other versions of NVDA? If so, please report their behaviors.

tested Chrome v140 with NVDA beta snapshot beta-52728,64493589 x86 - still with issue
tested any version of NVDA with Chrome v139 - no issue

### Windows version

Microsoft Windows 11 Pro version 10.0.26100 build 26100

### Name and version of other software in use when reproducing the issue

Chrome 140.0.7339.128 (official) (64 bit) (cohort: Stable) - Italian

### Other information about your system

Running italian version

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by hwf1324** [comment]
Duplicate of #18707

**Comment by albzan** [comment]
Thanks!
